docs/application/tizen-studio/release-notes/3-3-release-notes.md

```diff
@@ -0,0 +1,66 @@
+# Tizen Studio 3.3 Release Notes
+
+-   Release Date: July 26, 2019
+
+
+## IDE and Tools
+
+
+### Fixed Bugs
+-   Device Manager
+	-   Fixed the issue with logs not shown for Raspberry Pi board.
+-   IDE
+	-   Fixed the log view to work with TV devices in Tizen IDE.
+	-   App signing command incorrectly asking for Author and Distributor passwords has been fixed.
+	-   Running an application on emulator now launches the emulator if not in launched state already.
```

Comment:
Can we rephrase as:
Running an application on the emulator now launches the emulator automatically if the emulator is not in the launched state.

Please verify.